data/presets/roller_coaster/support.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
what about this one as an icon here: ![](https://raw.githubusercontent.com/openstreetmap/iD/b792a0106daf844af75373ef1b85176cecbe13f9/svg/roentgen/tower.svg)

```suggestion
{
    "icon": "roentgen-tower",
```